components/director/internal/graphql/scalars/helpers.go

```diff
@@ -0,0 +1,52 @@
+package scalars
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+func writeResponse(in interface{}, w io.Writer) error {
```

Comment:
Please rename this method to be more self-describing. This has nothing with writing the response.

Reply:
I've renamed it to `marshalAndWrite`, if you have a better name please give me a hint